docs/pt-br/strict.md

```diff
@@ -0,0 +1,26 @@
+# Strict Mode (ou Modo Estrito)
```

Comment:
Why not definitivelly translating "strict mode" to "modo estrito" on all documentation?